javascriptv3/example_code/cross-services/transcribe-streaming-app/README.md

```diff
@@ -0,0 +1,469 @@
+      LanguageCode: "en-US",
+      // The encoding used for the input audio. The only valid value is pcm.
+      MediaEncoding: "pcm",
+      // The sample rate of the input audio in Hertz.
+      MediaSampleRateHertz: 44100,
+      AudioStream: audioStream(),
+    });
+
+    // Send the speech stream to Amazon Transcribe.
+    const data = await transcribeClient.send(command);
+    console.log("Success", data.TranscriptResultStream);
+    for await (const event of data.TranscriptResultStream) {
+      for (const result of event.TranscriptEvent.Transcript.Results || []) {
+        if (result.IsPartial === false) {
+          const noOfResults = result.Alternatives[0].Items.length;
+          // Print results to browser window.
+          for (let i = 0; i < noOfResults; i++) {
+            console.log(result.Alternatives[0].Items[i].Content);
+            const outPut = result.Alternatives[0].Items[i].Content + " ";
+            const outputDiv = document.getElementById("output");
+            outputDiv.insertAdjacentHTML("beforeend", outPut);
+          }
+        }
+      }
+    }
+    console.log("Success. ", data);
+    client.destroy();
+  } catch (err) {
+    console.log("Error. ", err);
+  }
+};
+
+window.stopRecord = function () {
+  const micStream = new MicrophoneStream();
+  console.log("Recording stopped");
+  var record = document.getElementById("record");
+  var stop = document.getElementById("stopRecord");
+  record.disabled = false;
+  stop.disabled = true;
+  record.style.backgroundColor = "red";
+  micStream.stop();
+};
+
+window.translateText = async () => {
+  try {
+    const outPut = document.getElementById("output").innerHTML;
+    const data = await comprehendClient.send(
+      new DetectDominantLanguageCommand({ Text: outPut })
+    );
+    const langCode = data.Languages[0].LanguageCode;
+    try {
+      const selectedValue = document.getElementById("list").value;
+      const translateParams = {
+        Text: outPut,
+        SourceLanguageCode: langCode /* required */,
+        TargetLanguageCode: selectedValue /* required */,
+      };
+      const data = await translateClient.send(
+        new TranslateTextCommand(translateParams)
+      );
+      document.getElementById("translated").innerHTML = data.TranslatedText;
+    } catch (err) {
+      console.log("Error translating language. ", err);
+    }
+  } catch (err) {
+    console.log("Error detecting language of text. ", err);
+  }
+};
+
+window.clearTranscription = async () => {
+  document.getElementById("output").innerHTML = "";
+};
+
+// Helper function to send an email to user.
+window.sendEmail = async () => {
+  const toEmail = document.getElementById("email").value;
+  const outputDiv = document.getElementById("output").innerHTML;
+  const translatedDiv = document.getElementById("translated").innerHTML;
+  const fromEmail = "briangermurray@gmail.com";
```

Comment:
Probably don't want to include your actual email in this example...

Reply:
fixed